### PR TITLE
Fix wget2 with quickget (#878)

### LINKED ISSUE #878: Downloading fails when wget2 is used to replace wget
Fedora (https://fedoraproject.org/wiki/Changes/Wget2asWget) and other distros are starting to replace wget with wget2. Wget2 renames the "show-progress" option to "force-progress", causing downloads from quickget to fail.

### PR DESCRIPTION
**Pull request by lj3954**
Wget2 renames "show-progress" to "force-progress". This fixes the command when wget2 is installed. 
Fixes #878

### CODE REVIEW

File: quickget [hunk 1]
```diff
@@ -939,14 +939,20 @@ function web_get() {
       exit 1
     fi
 
+    if wget --help | grep -q Wget2; then
```
Comment (zen0bit) [inline quickget:942]: if wget -V | grep -q Wget2; then

Comment (lj3954) [reply quickget:942]: I think it's actually better to check whether wget2 is installed in the first place and use it if so; some distributions may not ship this package that symlinks /usr/bin/wget to /usr/bin/wget2. Wget2 is faster and should be used instead whenever available.

Comment (lj3954) [reply quickget]: Also, just remembered, the best way to check whether /usr/bin/wget leads to wget2 would be to use `readlink /usr/bin/wget` and check whether the output is equal to 'wget2'. That would be much faster than any other way, it only takes 2 milliseconds on my system compared to 6 with wget2's help and version menus.

**Comment by lj3954** [comment]
Wget2's output also does not work correctly for finding where a URL redirects. Instead, use curl and direct output to a nonexistent directory in order to only receive the final URL (/dev/null wouldn't work, since it's valid and you can download a file to it).

**Comment by 24fpsDaVinci** [comment]
thank you, this patch worked. maybe there could be a wget2 branch?

**Pull request closed without merging**